config/content-security-policy.js

```diff
@@ -21,82 +21,81 @@ module.exports = function csp(env) {
+      'https://www.google-analytics.com',
       // Bungie.net API
-      "https://www.bungie.net",
+      'https://www.bungie.net',
       // DTR Reviews API
-      "https://reviews-api.destinytracker.net",
-      "https://db-api.destinytracker.com",
-      // VendorEngrams.XYZ API
-      "https://api.vendorengrams.xyz"
+      'https://reviews-api.destinytracker.net',
+      'https://db-api.destinytracker.com'
     ],
     imgSrc: [
       SELF,
       // Webpack inlines some images
-      "data:",
+      'data:',
       // Bungie.net images
-      "https://www.bungie.net",
+      'https://www.bungie.net',
       // Google analytics tracking
-      "https://ssl.google-analytics.com",
-      "https://www.google-analytics.com",
-      "https://csi.gstatic.com",
+      'https://ssl.google-analytics.com',
+      'https://www.google-analytics.com',
+      'https://csi.gstatic.com',
       // OpenCollective backers
-      "https://opencollective.com",
+      'https://opencollective.com',
       // Twitter Widget
-      "https://syndication.twitter.com",
-      "https://platform.twitter.com",
-      "https://*.twimg.com/"
+      'https://syndication.twitter.com',
+      'https://platform.twitter.com',
+      'https://*.twimg.com/'
     ],
     fontSrc: [
       SELF,
       // Google Fonts
-      "https://fonts.gstatic.com"
+      'https://fonts.gstatic.com'
     ],
     childSrc: [
       SELF,
       // Google Login
-      "https://accounts.google.com",
-      "https://content.googleapis.com"
+      'https://accounts.google.com',
+      'https://content.googleapis.com'
     ],
     frameSrc: [
       // Google Login
-      "https://accounts.google.com",
-      "https://content.googleapis.com",
+      'https://accounts.google.com',
+      'https://content.googleapis.com',
       // Twitter Widget
-      "https://syndication.twitter.com/",
-      "https://platform.twitter.com/"
+      'https://syndication.twitter.com/',
+      'https://platform.twitter.com/'
     ],
     objectSrc: SELF,
     // Web app manifest
     manifestSrc: SELF
-  }
+  };
 
   // Turn on reporting to sentry.io on beta only
   if (env === 'beta') {
-    baseCSP.reportUri = "https://sentry.io/api/279673/csp-report/?sentry_key=1367619d45da481b8148dd345c1a1330"
-    baseCSP.connectSrc.push("https://sentry.io/api/279673/store/")
+    baseCSP.reportUri =
+      'https://sentry.io/api/279673/csp-report/?sentry_key=1367619d45da481b8148dd345c1a1330';
+    baseCSP.connectSrc.push('https://sentry.io/api/279673/store/');
   } else if (env === 'release') {
     // Allow release to load updated classified item definitions and images from beta
-    baseCSP.connectSrc.push("https://beta.destinyitemmanager.com");
-    baseCSP.imgSrc.push("https://beta.destinyitemmanager.com");
+    baseCSP.connectSrc.push('https://beta.destinyitemmanager.com');
+    baseCSP.imgSrc.push('https://beta.destinyitemmanager.com');
   }
 
   return builder({
     directives: baseCSP
   });
-}
+};
```

Comment:
Weird, it lived happily this long without the semicolon.

Reply:
Prettier is just doing its thing.